iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/twin/DesiredPropertiesTests.java

```diff
@@ -362,7 +360,7 @@ else if (!actualProperties.get(actualPropertyKey).equals(expectedProperties.get(
         }
     }
 
-    // This test is for the startTwinAsync/startTwinAsync API that takes the TwinPropertiesCallback rather than the TwinPropertyCallback
+    // This test is for the startTwinAsync/startTwinAsync API that takes the TwinPropertyCallBack rather than the TwinPropertyCallBack
```

Comment:
```suggestion
    // This test is for the startTwinAsync/startTwinAsync API that takes the TwinPropertiesCallback rather than the TwinPropertyCallback
```